### PR TITLE
Google.IAPDesktop version 2.18.631

### PR DESCRIPTION
**Pull request by ImJoakim**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16622)

### CODE REVIEW

**Comment by KaranKad** [comment]
This is not an official Google product. It needs different publisher name.

**Comment by jedieaston** [comment]
It shows up as Google in the ARP table and is written by people that work at Google. Even if they won't sell you support for it, they still "publish" it.

**Comment by KaranKad** [comment]
Oh! didn't know that.😅

File: manifests/g/Google/IAPDesktop/2.18.631/Google.IAPDesktop.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate
+
+PackageIdentifier: Google.IAPDesktop
+PackageVersion: 2.18.631
+PackageLocale: en-US
+Publisher: Google
```
Comment (jedieaston) [inline manifests/g/Google/IAPDesktop/2.18.631/Google.IAPDesktop.locale.en-US.yaml:6]:
```suggestion
Publisher: Google Inc

```

**Review by jedieaston — CHANGES_REQUESTED** [review]
One quick fix, then you should be good. Thanks!

File: manifests/g/Google/IAPDesktop/2.18.631/Google.IAPDesktop.locale.en-US.yaml (hunk 1 above)
Comment (ImJoakim) [reply manifests/g/Google/IAPDesktop/2.18.631/Google.IAPDesktop.locale.en-US.yaml:6]: Oh my bad, I was just following the previous layout. I was going to put "Google Inc" but I saw that "Google" was used previously.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**